File: client/Task2/Models/Study.swift
//
//  Study.swift
//  Task2
//
//  Created by skia mac mini on 5/8/24.
//

import Foundation

struct Study: Decodable {
    let id: Int
    let studyInstanceUID: String
    let studyID: String
    let studyDateTime: String
    let studyDescription: String
    let patientID: String
    let patientName: String
    let patientBirthDate: String
    let patientSex: String
    let referringPhysicianName: String?
    let numberOfSeries: Int
    let createdDateTime: String
    let lastModifiedDateTime: String
    let tag: Int?
    let memo: String?
}

File: server/DicomServer/Domains/Study.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DicomServer.Domain
{
    public struct Study
    {
        public int id { get; set; }
        public string studyInstanceUID { get; set; }
        public string studyID { get; set; }
        public string studyDateTime { get; set; }
        public string studyDescription { get; set; }
        public string patientID { get; set; }
        public string patientName { get; set; }
        public string patientBirthDate { get; set; }
        public string patientSex { get; set; }
        public string ? referringPhysicianName { get; set; }
        public int numberOfSeries { get; set; }
        public string createdDateTime { get; set; }
        public string lastModifiedDateTime { get; set; }
        public string ? tag { get; set; }
        public string ? memo { get; set; }

    }
}